ci/test-install/install_integration_test.cc

```diff
@@ -12,15 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <google/protobuf/text_format.h>
+#include <sstream>
+#include "bigtable/client/admin_client.h"
```

Comment:
Headers are in wrong order.

Reply:
That is the order that `clang-format(1)` picked.  If we change it, the automated build where we enforce formatting via clang-format would break.  Note that the current configuration for `clang-format` is the strict Google Style Guide, which is different than what is proposed in #98 .